client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { Gridicon } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { Icon, external } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { Site } from '../../types';
+
+import './style.scss';
+
+const ICON_SIZE = 24;
+
+interface Props {
+	selectedSite?: Site | undefined;
+	closeSitePreviewPane: () => void;
+}
+
+export default function SitePreviewPaneHeader( { selectedSite, closeSitePreviewPane }: Props ) {
+	if ( ! selectedSite ) {
+		return null;
+	}
+
+	// eslint-disable-next-line no-console
+	console.log( selectedSite );
```

Comment:
Don't forget to remove this console.log before merging this PR; it has the clause `eslint` disabled. 

Or better, remove it now.

Reply:
Removed in [2e167c1](https://github.com/Automattic/wp-calypso/pull/87760/commits/2e167c165473eec9400082d54b7c99dd51c64051)
Thanks!